OpenTabletDriver.Plugin/Tablet/Wheel/IRelativeWheelReport.cs

```diff
@@ -0,0 +1,13 @@
+namespace OpenTabletDriver.Plugin.Tablet.Wheel
+{
+    /// <summary>
+    /// An auxiliary report containing states of a wheel/ring input.
+    /// </summary>
+    public interface IRelativeWheelReport : IDeviceReport
+    {
+        /// <summary>
+        /// The relative wheel movement from its previous state, or null to indicate an absence of touch.
```

Comment:
```suggestion
        /// The relative wheel movement from its previous state, or 0 to indicate an absence of touch.
```
now that nullability is removed for that field

Reply:
I would instead suggest going back to `int?`.

I don't quite know why the original author originally went with a uint, but at least the HS610 can also detect that the finger is touching the wheel, even if its stationary.

So a report of zero would be the first touch before movement, and a report of null would indicate that the touch has ended.

Reply:
> now that nullability is removed for that field

Additionally i think that i need to reword the rest to state that it's not the absence of just touch, but the absence of movement in general

Reply:
That makes sense. You are welcome to add the suggestion directly, so that you can be credited with the change.

Reply:
> The relative wheel movement from its previous state, 0 to indicate an absence of movement or null for an absence of touch.

Seems more correct if i take in account every suggestions.